dms/views/dms_file.xml

```diff
@@ -315,6 +319,12 @@
                 <field name="is_locked" invisible="1" />
                 <field name="is_lock_editor" invisible="1" />
                 <field name="name" />
+                <field
+                    name="content"
+                    filename="name"
+                    widget="binary"
+                    string="Download"
+                />
```

Comment:
I think it would be better to add a button with an action to make the download.

Reply:
Thanks for your feedbacks!

I could change it to a url action and call something like:

`/web/content?id=14&amp;field=content&amp;model=dms.file&amp;filename_field=name&amp;download=true`

but I have to change it a bit to make it working with URL action, do you think this is right or you suggest a better approach? 

In URL action case I'm pretty sure that I can access record id by context, but I am not sure how pass the other dynamic value, which is "name" for "filename_field", have a suggestion for that?

Last question, is it really worth to transform this in an url action instead of simply using the widget? I think we might gain a bit more flexibility but I'm not sure if it's worth in this specific case.

Reply:
@victoralmau Removed `content` field in the kanban, waiting for feedback about this. 

Meanwhile I have made some experiments but I wasn't able to call the download from an URL action defined by xml because I didn't managed reach id and record name dynamically. 

I could add a type object button and return an url action by python, but (just my opinion) it would require much more code to obtain basically the same behaviour than I can obtain with the binary widget. 

Unless you mean to add a contextual action, I can see advantages on that (multiple downloads at same time) but it wasn't the scope of this PR.

What do you think?

Reply:
What do you think @pedrobaeza ?

Reply:
Not sure what do you want. Reading vertically, it seems you want a direct button to download without doing 2 clicks, so that's something reasonable, but how it's implemented is not so important in this case IMO. It should just work.

Reply:
Please see my concern on the bottom.